### PR TITLE
data: chaddyc

### PR DESCRIPTION
**Pull request by chaddyc**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/7113"></a>

### CODE REVIEW

File: data/chaddyc.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
+  "name": "Chad Crouch",
+  "bio": "Open Source Enthusiast | Software/DevOps Engineer | Tech Blogger @ Opensource Geeks",
+  "links": [
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/fossgeek",
+      "icon": "FaTwitter"
+    }
```
Comment (khushalarora) [inline data/chaddyc.json:9]: Please add a **'Comma'** at the end of this line :+1: 
```Suggestion
    },
```

File: data/chaddyc.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    {
+      "name": "Follow me on Github",
+      "url": "https://github.com/chaddyc",
+      "icon": "FaGithub"
+    }
```
Comment (khushalarora) [inline data/chaddyc.json:14]: Also, add a **'Comma'** here :+1: 
```Suggestion
    },
```

**Review by khushalarora — CHANGES_REQUESTED** [review]
Hey @chaddyc, thank you for contributing :tada:

**Review by SaraJaoude — APPROVED** [review]
looks good

**Pull request merged**